actions/get-compatible-stacks/entrypoint/main.go

```diff
@@ -75,5 +75,7 @@ func main() {
 	}
 
 	log.Println("Output: ", string(output))
-	fmt.Printf("::set-output name=compatible-stacks::%s\n", string(output))
+	actions.Outputs{
```

Comment:
I am ambivalent on pulling in the pipeline-builder library for this purpose.

On the one hand, it's fairly convenient to reuse the pre-existing logic. On the other hand, it's a fairly big increase in the surface area.

Ultimately, this is an action, not production code, so I don't think I have any real objections.